docs/proposals/scalability-testing.md

```diff
@@ -0,0 +1,78 @@
+
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/docs/proposals/scalability-testing.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+## Background
+
+We have a goal to be able to scale to 1000-node clusters by end of 2015.
+However, running regression tests on 1000-node clusters or requiring developers
```

Comment:
We've been planning to oversubscribe using a "glidein" practice we've used on other clustering systems to simulate scale.  The practice is well established in scientific clustering (pbs, globus, condor, etc), and depends heavily on https://github.com/GoogleCloudPlatform/kubernetes/pull/9976 being functional.  

In this scenario 26 x 40 core machines appear as 1040 machines.